packages/sync/src/Actions.php

```diff
@@ -355,6 +356,12 @@ public static function do_initial_sync() {
 			return false;
 		}
 
+		// Don't start new sync if a full sync is in process.
+		$full_sync_module = Modules::get_module( 'full-sync' );
+		if ( $full_sync_module && ( $full_sync_module->is_started() && ! $full_sync_module->is_finished() ) ) {
```

Comment:
Seems like there's no need for the extra parens:

```suggestion
		if ( $full_sync_module && $full_sync_module->is_started() && ! $full_sync_module->is_finished() ) {
```